examples/route-mirror/verify.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+export NAME=route-mirroring
+export PORT_PROXY="${FRONT_PROXY_PORT_PROXY:-11820}"
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+
+run_log "Static mirror cluster: localhost:${PORT_PROXY}/service/1"
+wait_for 20 bash -c "responds_with \"Hello from behind Envoy (service 1)!\" http://localhost:${PORT_PROXY}/service/1"
+
+docker-compose logs service1 | grep --quiet "Host: localhost:${PORT_PROXY}"
```

Comment:
```suggestion
run_log "View logs for the request mirrored by request header"
docker-compose logs service1 | grep --quiet "Host: localhost:${PORT_PROXY}"
```


... same with steps below ...